distribution/deb/pom.xml

```diff
@@ -5,13 +5,13 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.elasticsearch.distribution</groupId>
-        <artifactId>elasticsearch-distribution</artifactId>
+        <artifactId>distributions</artifactId>
         <version>2.1.0-SNAPSHOT</version>
     </parent>
 
     <groupId>org.elasticsearch.distribution.deb</groupId>
     <artifactId>elasticsearch</artifactId>
-    <name>Elasticsearch DEB Distribution</name>
+    <name>Distribution: Debian</name>
```

Comment:
Minor quibble: Distribution: deb

Its to make deb packages, not to support debian flavored distributions.